Kalela/Level-Up
# 155820624: DetailLayoutActivity

# What does this PR do?
- Creates the layout for the DetailLayout activity.
- Allows DetailLayoutActivity to be launched from the MainActivity.
### Screenshot
![ezgif com-optimize](https://user-images.githubusercontent.com/33759757/42091753-31ea4d22-7baf-11e8-90af-247985603e04.gif)

### How should this be manually tested?
1. Clone the repo to your local machine.
2. Open the project with Android Studio.
3. Run an emulator or adb connection and run the app.
4. Tap on one of the developers in the main page to see the details activity.

### What are the relevant pivotal tracker stories?
- 155820645
Create layout file for DetailActivity
- 155820624
Launch the DetailActivity from the MainActivity
